airflow/providers/hashicorp/secrets/vault.py

```diff
@@ -150,7 +156,29 @@ def get_conn_uri(self, conn_id: str) -> Optional[str]:
         :param conn_id: connection id
         :type conn_id: str
         """
-        secret_path = self.build_path(self.connections_path, conn_id)
+        response = self._get_secret(self.connections_path, conn_id)
+        return response.get("conn_uri") if response else None
+
+    def get_variable(self, key: str) -> Optional[str]:
+        """
+        Get Airflow Variable from Environment Variable
+
+        :param key: Variable Key
+        :return: Variable Value
+        """
+        response = self._get_secret(self.variables_path, key)
+        return response.get("value") if response else None
```

Comment:
@kaxil Why get `response.get("value")` instead of `response[key]`?
Specifically, why 
```
variables
 |- foo
 |    |- value = some_value
└ bar
     └ value = another_value
```
Not
```
variables
 |- foo = some_value
└ bar = another_value
```
This makes the structure simple.

Reply:
Hi @jacobhjkim it was mainly because we wanted to store each Variable as a separate secret (to have proper access control / avoid limits on a secret) and the fact that we use KV store

Reply:
I see, thanks for the answer 👍